src/test/evo_netinfo_tests.cpp

```diff
@@ -28,7 +28,7 @@ const std::vector<std::pair</*input=*/std::string, /*expected_ret=*/NetInfoStatu
     // Port greater than uint16_t max
     {"1.1.1.1:99999", NetInfoStatus::BadInput},
     // Only IPv4 allowed
-    {"[2606:4700:4700::1111]:9999", NetInfoStatus::BadType},
+    {"[2606:4700:4700::1111]:9999", NetInfoStatus::BadInput},
```

Comment:
does it mean that there's no more unit test for `BadType`? I don't see any

Reply:
Not in this PR since serialization-based checks are limited to `CService` compatibility so far but they're there in #6666 as input validation checks.

https://github.com/dashpay/dash/blob/e2f1a8f94a932703e5acae4ca33088e4dc7359c0/src/test/evo_netinfo_tests.cpp#L34-L39